mobile/src/main/java/org/openhab/habdroid/ui/PreferencesActivity.kt

```diff
@@ -109,7 +110,7 @@ class PreferencesActivity : AbstractBaseActivity() {
         protected abstract val titleResId: Int
 
         protected val parentActivity get() = activity as PreferencesActivity
-        protected val prefs get() = preferenceScreen.sharedPreferences
+        protected val prefs get() = preferenceScreen.sharedPreferences as SharedPreferences
```

Comment:
Is this for getting rid of nullability? Can shortcut with `!!` instead of cast in that case.

Reply:
Kotlin throws a lint issue here, because it doesn't know for sure if it receives `SharedPreferences` or `SharedPreferences?`.

Reply:
That's what I guessed. As I said, can shortcut using the `!!` operator in that case.

Reply:
Done.